frontend/src/app/testResults/uploads/specificSchemaBuilder.ts

```diff
@@ -584,121 +644,40 @@ export const specificSchemaBuilder = (facilityId: string | null) => {
-                  "<mark><code>Military Accommodation</code></mark> or <mark><code>224683003</code></mark>",
-                  "<mark><code>Hospice</code></mark> or <mark><code>284546000</code></mark>",
-                  "<mark><code>Hostel</code></mark> or <mark><code>257628001</code></mark>",
-                  "<mark><code>Sheltered Housing</code></mark> or <mark><code>310207003</code></mark>",
-                  "<mark><code>Penal Institution</code></mark> or <mark><code>57656006</code></mark>",
-                  "<mark><code>Religious Institutional Residence</code></mark> or <mark><code>285113009</code></mark>",
-                  "<mark><code>Work (environment)</code></mark> or <mark><code>285141008</code></mark>",
-                  "<mark><code>Homeless</code></mark> or <mark><code>32911000</code></mark>",
-                ],
-                description: [
-                  "If the resident congregate setting is “Y” or “Yes,” then provide residence type",
-                ],
-              },
-              {
-                name: "Hospitalized",
-                colHeader: "hospitalized",
-                required: false,
-                requested: true,
-                acceptedValues: [
-                  "<mark><code>Y</code></mark> or <mark><code>YES</code></mark>",
-                  "<mark><code>N</code></mark> or <mark><code>NO</code></mark>",
-                  "<mark><code>U</code></mark> or <mark><code>UNK</code></mark>",
-                ],
-                format:
-                  "If the patient tested was admitted to a hospital for treatment.<br/>Use one of the accepted values listed below.",
-              },
-              {
-                name: "Intensive care unit",
-                colHeader: "icu",
-                required: false,
-                requested: true,
-                acceptedValues: [
-                  "<mark><code>Y</code></mark> or <mark><code>YES</code></mark>",
-                  "<mark><code>N</code></mark> or <mark><code>NO</code></mark>",
-                  "<mark><code>U</code></mark> or <mark><code>UNK</code></mark>",
+            tabs: [
+              {
+                title: MULTIPLEX_DISEASES.COVID_19,
+                slug: MULTIPLEX_DISEASES.COVID_19,
+                items: [
+                  aoeDocumententationItems.pregnant,
+                  aoeDocumententationItems.employed_in_healthcare,
+                  aoeDocumententationItems.symptomatic_for_disease,
+                  aoeDocumententationItems.illness_onset_date,
+                  aoeDocumententationItems.resident_congregate_setting,
+                  aoeDocumententationItems.residence_type,
+                  aoeDocumententationItems.hospitalized,
+                  aoeDocumententationItems.icu,
+                ],
+              },
+              {
+                title: MULTIPLEX_DISEASES.HIV,
+                slug: MULTIPLEX_DISEASES.HIV,
+                items: [
```

Comment:
In this PR are we addressing the HIV AOE questions or just adding the disease specific AOE tabs?

Because if we want the guide to reflect what is required we should add genders of sexual partners.

Let me know what you think! 👀